<map id="sysio::http_plugin::handle_exception" name="sysio::http_plugin::handle_exception">
<area shape="rect" id="Node000001" title=" " alt="" coords="192,41,322,82"/>
<area shape="rect" id="Node000002" href="$classsysio_1_1http__plugin.html#ac1a2d5844cede3082aade8a1aeb798a2" title=" " alt="" coords="13,5,136,46"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="176,48,136,40,137,35,177,43"/>
<area shape="rect" id="Node000003" href="$structsysio_1_1trace__api__rpc__plugin__impl.html#a8d5216121c2ae237cf5bb675b66aef1f" title=" " alt="" coords="5,70,144,126"/>
<area shape="poly" id="edge2_Node000001_Node000003" title=" " alt="" coords="177,80,145,87,143,81,176,75"/>
</map>
